packages/gatsby/src/gatsby-node.js

```diff
@@ -0,0 +1,20 @@
+exports.onCreateWebpackConfig = ({ plugins, actions }) => {
+  actions.setWebpackConfig({
+    plugins: [
+      plugins.define({
+        __SENTRY_RELEASE__: JSON.stringify(
```

Comment:
@dcramer I am confused why you call `JSON.stringify` on something that should just be a string already.

Reply:
The define plugin does a string replace, so it will insert the literal value of whatever you give it (which means it would be an unquoted string)